.circleci/config.yml

```diff
@@ -193,6 +209,12 @@ jobs:
     docker:
       - image: node:14.7
     steps:
+      - run:
+          name: Load salesforce_env file
+          command: |
+            cat .salesforce_env_prod | while read line; do
+              export $line
```

Comment:
GitHub has a magic file you can write to, in order to set env vars (`$GITHUB_ENV`), so you might be able to just cat into that instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
ah nevermind, this is for Circle CI